frontend/src/metabase/admin/settings/components/CloudPanel/CloudPanel.tsx

```diff
@@ -104,6 +105,10 @@ export const CloudPanel = ({
           {migration && migrationState === "error" && (
             <MigrationError migration={migration} />
           )}
+
+          {createCloudMigrationResult.isError && (
+            <MigrationCreationError error={createCloudMigrationResult.error as MigrationCreationErrorProps} />
```

Comment:
We generally try to avoid typecasting when possible. I can take a look and see if I can resolve the type issues if you'd like

Reply:
My editor was saying this was a `unknown`, which really surprised me. But yeah feel free to, or I will later.

Reply:
Saw another component also using a [result error](https://github.com/metabase/metabase/blob/057cb3f38677fa6bfa14408f7c6919422d8856c1/frontend/src/metabase/admin/performance/components/StrategyEditorForDatabases.tsx#L125). Used the same casting (`any`) on the props.